modules/storages/spec/contracts/storages/storages/shared_contract_examples.rb

```diff
@@ -116,6 +116,18 @@
         include_examples 'contract is invalid', host: :url
       end
 
+      context 'when host is an unsafe IP' do
+        let(:storage_host) { 'http://172.16.193.146' }
+
+        include_examples 'contract is invalid', host: :url_not_secure_context
+      end
+
+      context 'when host is an unsafe hostname' do
+        let(:storage_host) { 'http://nc.openproject.com' }
+
+        include_examples 'contract is invalid', host: :url_not_secure_context
+      end
```

Comment:
Can we add a positive and negative example with ipv6? This has two reasons: 

- we check if it works, also for future changes on the validator
- there are a minority of users, who check tests for usage examples. So they could check, how an ipv6 addr can be provided (it must have the brackets, mustn't it?)

UPDATE:
just saw, that we have those test cases directly for the validator. this suffices in my opinion